tests/sqllogictests/suites/base/03_common/03_0039_insert_into_bitmap

```diff
@@ -27,4 +27,16 @@ SELECT id, to_string(v) FROM t1 order by id
 6 9,10
 
 statement ok
-DROP DATABASE db_bitmap
\ No newline at end of file
+CREATE TABLE IF NOT EXISTS t2(a int not null, b bitmap not null) Engine = Fuse
+
+statement ok
+insert into t2 (a) values (1),(2);
+
+query IT
+SELECT a, to_string(b) FROM t2
+----
+1 0,1
+2 1,2

```

Comment:
`to_string(b)` should return an empty string, as the default value is empty bitmap